website/content/en/docs/building-operators/ansible/quickstart.md

```diff
@@ -1,381 +1,96 @@
 ---
-title: Ansible Operator Quickstart
-linkTitle: Quickstart
+title: Ansible Operator QuickStart
+linkTitle: QuickStart
 weight: 2
 ---
+## Prerequisites
```

Comment:
The ## Prerequisites is what is required to follow up the quick-start which means:

- [docker][docker_tool] version 17.03+. (docker because we need to build the image)
- [kubectl][kubectl_tool] version v1.11.3+. (because we need to apply the things in the cluster)
- [operator-sdk][operator_install] v1.0.0+ (here is the link to install SDK cli tool )
- Access to a Kubernetes v1.11.3+ cluster. (a K8S cluster)

So, I am not sure if I got your suggestion here. Because : 

> This section should be removed, instead link to installation

- The install does not define the pre-requirements to run the quick start
- As a pre-requirement, we are already linking the install

> (It is missing the ansible specific prereqs)

What ansible specific is required to run the commands in the quick start? I understand that the additional is not required here at all.

Reply:
https://sdk.operatorframework.io/docs/ansible/installation/#additional-prerequisites

Reply:
The ansible installation prerequisites are left out https://sdk.operatorframework.io/docs/ansible/installation/#additional-prerequisites 

kubectl, docker, and access to the cluster are all fine to stay.